contrib/mod_sftp/fxp.c

```diff
@@ -5162,7 +5165,7 @@ static int fxp_handle_ext_posix_rename(struct fxp_packet *fxp, char *src,
     src, dst);
 
   cmd = fxp_cmd_alloc(fxp->pool, "RENAME", args);
-  cmd->cmd_class = CL_MISC|CL_SFTP;
+  cmd->cmd_class = CL_MISC|CL_WRITE|CL_SFTP;
```

Comment:
Here we try to align the command classes for SFTP requests more closely with the command classes assigned to their FTP equivalents, where possible.